orchestrator/api/dashboard_integration.py

```
@@ -95,57 +87,45 @@ async def track_agent_execution(
-                knowledge_transfers=knowledge_transfers
-            )
-    except Exception as e:
-        logger.error(f"Error tracking learning progress: {e}")
+async def broadcast_dashboard_update(update_type: str, data: dict):
+    """Broadcast update to dashboard - now via Redis pub/sub only"""
+    if redis_client:
+        try:
+            import json
+            message = json.dumps({"type": update_type, "data": data})
+            await redis_client.publish("dashboard_updates", message)
+        except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)